### PR TITLE
Enable target s390x and ppc64le on x86-86 host on SUSE

### LINKED ISSUE #361: support for architecture s390, ppcle on Suse with docker story
We'd like to enable to produce rpm for ppc64le, s390x on x86_64 host for SUSE package. With update the spec file and using the command `rpmbuild  --target "$target" -bb "$spec"`  will get the error message  https://github.com/adoptium/installer/compare/master...sophia-guo:plats1?expand=1

```
+ rpmbuild  --target x86_64 -bb /home/builder/build/generated/packaging/temurin-17-jdk.spec
error: File /home/builder/rpmbuild/SOURCES/OpenJDK17U-jdk_x64_linux_hotspot_17.0.1_12.tar.gz: No such file or directory
Building target platforms: x86_64
```

Which works fine as long as I remove the --target option. Looks like with SUSE package --target option of rpmbuild doesn't work?

### PR DESCRIPTION
**Pull request by sophia-guo**
Enable target s390x and ppc64le on x86-86 host on SUSE

Enable SRPM and copy SPRM to destination Directory

close #361 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

File: linuxNew/jdk/suse/src/main/packaging/temurin/8/temurin-8-jdk.spec [hunk 1]
```diff
@@ -220,5 +229,5 @@ fi
 %{prefix}
 
 %changelog
-* Tue Aug 31 2021 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 8.0.302.0.0.8-1.adopt0
+* Fri Aug 31 2021 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 8.0.302.0.0.8-1.adopt0
```
Comment (jerboaa) [inline linuxNew/jdk/suse/src/main/packaging/temurin/8/temurin-8-jdk.spec:232]: Aug 31 was actually a `Tue`. Produces a `bogus date in changelog` warning.

Comment (sophia-guo) [reply linuxNew/jdk/suse/src/main/packaging/temurin/8/temurin-8-jdk.spec]: Updated. Though this log will need to be updated to 8u312-b07 version. The PR https://github.com/adoptium/installer/pull/357/files didn't update changelog. Will do that in a separate PR.

**Comment by sophia-guo** [comment]
https://ci.adoptopenjdk.net/view/work-in-progress/job/adoptium-packages-linux-pipeline/57/console
https://ci.adoptopenjdk.net/view/work-in-progress/job/adoptium-packages-linux-pipeline/56/console
https://ci.adoptopenjdk.net/view/work-in-progress/job/adoptium-packages-linux-pipeline/55/console

jdk package can be successfully generated.

**Comment by jerboaa** [comment]
@sophia-guo This introduces more PR test failures. I suggest to address #375 (e.g. by skipping testing on those for now) and/or #378

**Review by jerboaa — APPROVED** [review]
Nice work!

**Pull request merged**